src/components/theme/ContentMetadataTags/ContentMetadataTags.jsx

```diff
@@ -116,6 +94,20 @@ const ContentMetadataTags = (props) => {
           />
         )}
         <meta name="twitter:card" content="summary_large_image" />
+        <meta
+          property="twitter:url"
+          content={seo_canonical_url || toPublicURL(props.content['@id'])}
+        />
+        <meta property="twitter:site" content={'@ericof'} />
```

Comment:
I would love to get the credit, but this needs to be configured somewhere else :-D

Reply:
LOL!

Reply:
@ericof We don't have that data, right?